app.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const app = express();
+const endpointsJson = require("./endpoints.json");
+
+app.use(express.json());
```

Comment:
Please remove this until you need it (for a post or patch) - you should only write the code needed to complete the current ticket.